server_addon/houdini/client/ayon_houdini/api/pipeline.py

```diff
@@ -85,10 +92,9 @@ def install(self):
             # initialization during start up delays Houdini UI by minutes
             # making it extremely slow to launch.
             hdefereval.executeDeferred(shelves.generate_shelves)
-
-        if not IS_HEADLESS:
-            import hdefereval # noqa, hdefereval is only available in ui mode
             hdefereval.executeDeferred(creator_node_shelves.install)
+            if os.environ.get("AYON_WORKFILE_TOOL_ON_START"):
```

Comment:
Whenever `AYON_WORKFILE_TOOL_ON_START` env var is `"0"` this will also evaluate as True. We should use `from ayon_core.lib import env_value_to_bool`

```suggestion
            if env_value_to_bool("AYON_WORKFILE_TOOL_ON_START"):
```

Reply:
https://github.com/ynput/ayon-core/pull/682/commits/583bc8f86b596ed0e6fc27b97e46b20d5b9929b5